controllers/clustergroupupgrade_controller.go

```diff
@@ -848,6 +848,12 @@ func (r *ClusterGroupUpgradeReconciler) processManagedPolicyForUpgradeContent(
 			return err
 		}
 
+		// If the policy types did not return with useful data then continue to the next policy
+		// We don't want to create empty fields in the map with null values as that is confusing
+		if policyTypes == nil || len(policyTypes) == 0 {
```

Comment:
Minor comment. len() in go is actually nil safe: https://go.dev/tour/moretypes/12